p4src/main.p4

```diff
@@ -346,6 +349,23 @@ control PreQosPipe (inout parsed_headers_t    hdr,
         }
     }
 
+    action set_default_tc(tc_t tc) {
+        local_meta.tc = tc;
+    }
+
+    table default_tc {
```

Comment:
remove default TC, and enforce pfcp to always provide TC in terminations actions